vendor/tests/test_processor.py

```diff
@@ -1256,6 +1256,120 @@ def test_process_payment_transaction_fail_invalid_expiration(self):
         self.assertFalse(self.processor.transaction_submitted)
         self.assertEquals(InvoiceStatus.CART, self.processor.invoice.status)
 
+    def test_process_payment_fail_cvv_no_match(self):
+        """
```

Comment:
@fahzee1 @rhimmelbauer is this making an api call?

Reply:
Yes in the stripe test account